### ISSUE TITLE
Missing summary information

### ISSUE THREAD

**Issue by paapereira** [podverse/podverse-rn#1110]
**Describe the bug**
The summary information is lacking when comparing to AntennaPod for example. It's a custom RSS feed, namely Linux Unplugged.

**To Reproduce**
1. Check the summary of an episode from a custom feed in AntennaPod for example
2. Check the summary of that episode in Podverse
3. See the difference

**Expected behavior**
All the summary information should be available.

**Screenshots**
![antennapod](https://user-images.githubusercontent.com/2096208/180594976-f8fcfb29-c4f2-40da-81d6-040bc6996515.jpg)
![podverse](https://user-images.githubusercontent.com/2096208/180594986-0a00c2ad-5c48-47b3-a599-89ddb237367d.jpg)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy A40
 - OS: Android 11
 - App Version: 4.5.13

**Comment by paapereira** [comment]
I checked the public Linux Unplugged feed and the summary is also lacking so it's not a custom rss issue.

**Comment by mitchdowney** [comment]
Related https://podcastindex.social/@mitch/108701589801516039

**Comment by mitchdowney** [comment]
As a quick fix, I'm having our parser use the <itunes:summary> tag in the RSS feed as our Episode.description column value. If <itunes:summary> is not available, then it will fallback to the <description> tag.

Maybe not the best fix...but we'll see if it's passable. Basically in our Episode schema, `subtitle` means a short summary, and `description` means a full summary.

I just deployed the update to our feed parsers. The fixed (longer) summaries won't appear until we re-parse a feed, so the pages won't correct themselves until a new episode is released.

I also made a corresponding parser change for custom RSS feeds on mobile. The change will be in 4.5.14.

https://github.com/podverse/podverse-api/pull/497/files#diff-153165a1b6d603977febcc95a3218fa4a84eaccc5cda185e46165534a37629b3R802

https://github.com/podverse/podverse-rn/commit/7203dbd9c8f1113f7035542208fca42cc01bbf4e

<img width="1284" alt="image" src="https://user-images.githubusercontent.com/2262919/180641090-70709ee2-3c81-43de-b8ad-63e0b3cb3520.png">
